assets/js/googlesitekit/datastore/user/user-info.test.js

```diff
@@ -364,5 +364,53 @@ describe( 'core/user userInfo', () => {
 				expect( global[ userDataGlobal ] ).not.toEqual( undefined );
 			} );
 		} );
+
+		describe( 'getAccountChooserURL', () => {
+			it( 'throws an error if a destinationURL is not given', () => {
+				expect( () => {
+					registry.select( CORE_USER ).getAccountChooserURL();
+				} ).toThrow( 'destinationURL is required' );
+			} );
+
+			it( 'returns the encoded destination url with the email appended', async () => {
+				global[ userDataGlobal ] = userData;
+
+				const testURL = 'test destination url';
+				const userEmail = global[ userDataGlobal ].user.email;
+
+				registry.select( CORE_USER ).getAccountChooserURL( testURL );
+
+				await subscribeUntil(
+					registry,
+					() =>
+						registry
+							.select( CORE_USER )
+							.getAccountChooserURL( testURL ) !== undefined
+				);
+
+				const accountChooserURL = registry
+					.select( CORE_USER )
+					.getAccountChooserURL( testURL );
+
+				expect( accountChooserURL ).toBe(
+					`${ encodeURIComponent(
+						testURL
+					) }&Email=${ encodeURIComponent( userEmail ) }`
+				);
```

Comment:
This test isn't correct, because the ACs show this should include the account chooser URL and should be placing the "destination URL" in a `continue=` query param:

<img width="833" alt="CleanShot 2022-07-15 at 12 49 14" src="https://user-images.githubusercontent.com/90871/179217501-5b9c2ee6-e67d-4ca3-b3a6-b57cd77667dc.png">

The right `expect` call here would be:

```suggestion
				expect( accountChooserURL ).toBe(
					`https://accounts.google.com/accountchooser?continue=${ encodeURIComponent(
						testURL
					) }&Email=${ encodeURIComponent( userEmail ) }`
				);
```

(Note: this will fail, but that should be the value here and the tests and code should be updated to match 🙂 

Otherwise, the URL we're creating and expecting here would be: `test destination url&Email=myemail@test.com`.